docs/book/shipments.rst

```diff
@@ -4,49 +4,139 @@
 Shipments
 =========
 
-...
+A **Shipment** is a representation of a shipping request for an Order. Sylius supports multiple shipments per one Order.
+
+How to create a Shipment programmatically?
```

Comment:
I'd remove this part, as it is not that simple, it works, but the structure is not correct. We should use the processor to create shipments for an order, but we can document it later.